Package.swift

```diff
@@ -0,0 +1,14 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "VersionUpdater",
+    platforms: [.iOS(.v11)],
+    products: [.library(name: "VersionUpdater", targets: ["VersionUpdater"])],
+    targets: [
+        .target(name: "VersionUpdater", path: "Sources", resources: [
+            .process("VersionUpdater.bundle")
```

Comment:
`VersionUpdater.bundle` is at top level, above Sources, so maybe `../VersionUpdater.bundle` is needed. Also, use [copy](https://developer.apple.com/documentation/swift_packages/resource/3516880-copy) if process doesn't work

### Info
https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package

### Affected code
```swift
let bundlePath = Bundle.main.path(
            forResource: "VersionUpdater",
            ofType: "bundle",
            inDirectory: "Frameworks/VersionUpdater.framework"
            )!
```

### Doc's recommended usage in code
```swift
let settingsURL = Bundle.module.url(forResource: "settings", withExtension: "plist")
```

Reply:
A good resource: https://developer.apple.com/documentation/swift_packages/localizing_package_resources

Reply:
Based on testing and [this thread](https://developer.apple.com/forums/thread/650158), bundle accessors for SPM and other package managers is mutually exclusive, which means I will maintain this as a separate fork and this PR cannot be merged.
 

Bundle accessor | SPM | Cocoapods/Carthage
-|-|-
`Bundle(for: Self.self)` | 💥 | ✅
`Bundle.module` | ✅ | 💥

`Bundle.module` is essentially SPM magic...

https://developer.apple.com/forums/thread/650158?answerId=614513022#614513022